network/paths.yaml

```
@@ -206,19 +298,19 @@ WatchdogPatch:
           $ref: schemas.yaml#/Watchdog
   responses:
     '200':
-      description: Wi-Fi Watchdog config updated
+      description: Network Watchdog config updated
     '401':
       $ref: ../common/responses.yaml#/Unauthorized
     '500':
       $ref: ../common/responses.yaml#/InternalServerError
   security:
   - BasicAuth: []
-  summary: Configure Wi-Fi Watchdog
+  summary: Configure Network Watchdog
   description: |
     Configure the Wi-Fi watchdog, which runs whenever the Wi-Fi station (`sta`)
```

```suggestion
    Configure the network watchdog, which runs whenever the Wi-Fi station (`sta`)
```

also, change the `which runs whenever the Wi-Fi station` to the new behavior